src/query/models/strconv/int_printer.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018 Uber Technologies, Inc.
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
Not sure how this got detected as a moved file, gut I deleted the resolver and added this file instead